core/vibes/soul/primitives/navigation/index.tsx

```diff
@@ -92,6 +96,9 @@ interface Props<S extends SearchResult> {
   locales?: Locale[];
   activeLocaleId?: string;
   localeAction?: LocaleAction;
+  currencies?: Currency[];
+  activeCurrencyCode?: string;
```

Comment:
I think we should follow the same pattern as the locale above and use the nomenclature of `activeCurrencyId`. The interface for `Currency` would be this instead:
```ts
interface Currency {
  id: string; // The currency code
  label: string // Separates the display value from the identifier. For now, could be the same value as `id`.
}
```

Reply:
Agreed!